data-prepper-core/src/test/java/com/amazon/dataprepper/pipeline/PipelineTests.java

```diff
@@ -58,23 +58,23 @@ public void testPipelineState() {
     }
 
     @Test
-    public void testPipelineStateWithPrepper() {
+    public void testPipelineStateWithProcessor() {
```

Comment:
I think you should keep the existing test (`testPipelineStateWithPrepper`) and add the new one. This way the tests are testing both a `Prepper` and a `Processor`.

Reply:
I added a testPipelineStateWithPrepper, please confirm my implementation was what you intended. https://github.com/sbayer55/data-prepper/blob/rename-prepper-to-processor/data-prepper-core/src/test/java/com/amazon/dataprepper/pipeline/PipelineTests.java#L62

Reply:
Yes, it is. Thank you.